### PR TITLE
Added skyux documentation for list view grid columns

### PR DESCRIPTION
**Pull request by Blackbaud-CaseyKour**

### CODE REVIEW

**Comment by Blackbaud-LuisBello** [comment]
can we get a screenshot for what the updated demo page looks like?

**Comment by Blackbaud-CaseyKour** [comment]
updated sky documentation

File: src/app/components/list-view-grid/index.html [hunk 1]
```diff
@@ -72,6 +72,65 @@
...
+    >
+      Boolean determines if the column is draggable.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+        propertyName="hidden"
+        defaultValue=""
+        isOptional="true"
+        defaultValue="false"
+    >
+      Boolean determines if the column is displayed.
```
Comment (Blackbaud-LuisBello) [inline src/app/components/list-view-grid/index.html]: "Boolean determines if the column is displayed.  (default: false)" this reads as if the column does not display by default. I would change the wording on this description for the hidden property

File: src/app/components/list-view-grid/index.html [hunk 2]
```diff
@@ -72,6 +72,65 @@
...
+        propertyName="id"
+        defaultValue=""
+    >
+      Specify a unique identifier for the column. Is required if <code>field</code> is not specified.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+        propertyName="field"
+        defaultValue=""
+    >
+      Specify a name to identify the column. Is required if <code>id</code> is not specified.
```
Comment (Blackbaud-KristinaDurivage) [inline src/app/components/list-view-grid/index.html]: This should have information about how it binds the column to data, right @Blackbaud-LuisBello ?

Comment (Blackbaud-LuisBello) [inline src/app/components/list-view-grid/index.html]: yeah, that would be helpful

**Pull request merged**